utils/utils-lib/src/test_constants.rs

```diff
@@ -1,3 +1,3 @@
 /// Automatically generated by ROOT/scripts/rust/branch_ancestor.sh
```

Comment:
`automatically generate....` so this change should be made elsewhere?

Reply:
No, it doesn't matter, this file is always re-generated when compiling, but it's checked in so I thought might as well keep it up to date for when browsing on github.